nessie/src/test/java/org/apache/iceberg/nessie/TestNessieTable.java

```diff
@@ -400,7 +405,7 @@ private void validateRegister(TableIdentifier identifier, String metadataVersion
 
   @Test
   public void testRegisterTableWithGivenBranch() {
-    List<String> metadataVersionFiles = metadataVersionFiles(TABLE_NAME);
+    List<String> metadataVersionFiles = metadataVersionFiles(tableNameWithUUID(catalog.loadTable(TABLE_IDENTIFIER)));
```

Comment:
It seems odd to use the location to get a directory name, but then add the rest of the location back to list files. Why not just update `metadataVersionFiles` and `manifestFiles` to accept a full path?

Reply:
ok. Updated now.